src/app/providers/data.service.ts

```diff
@@ -34,6 +36,15 @@ export class DataService {
         this._activeWindowsSource.next(this.getActiveWindow());
     }
 
+    public getWindows():Observable<Array<Window>>{
```

Comment:
did you try using the other method i mentioned during the meeting before implementing this? you should be able to subscribe to the activeWindows$ data member from another component and achieve the same result. see the attached image from the gridcomponent. 

![window observable](https://user-images.githubusercontent.com/9039337/32761197-0c01dd2a-c8c0-11e7-9108-f56ff58d2209.png)